api/v1/db/oracledb/game-dao.js

```diff
@@ -0,0 +1,61 @@
+const appRoot = require('app-root-path');
+const _ = require('lodash');
+
+
+const { serializeGames } = require('../../serializers/games-serializer');
+
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+const mergeRawGames = (rawGames) => {
+  const groupedRawGames = _.groupBy(rawGames, 'GAME_ID');
+  const mergedRawGames = _.map(groupedRawGames, (gameMetaDataArray) => {
+    if (gameMetaDataArray[0].CARD_NUMBER == null) {
+      gameMetaDataArray[0].tableCards = [];
+    } else {
+      gameMetaDataArray[0].tableCards = _.map(gameMetaDataArray, (data) => {
+        return {
+          cardNumber: data.CARD_NUMBER,
+          cardSuit: data.SUIT,
+        };
+      });
+    }
+    return (gameMetaDataArray[0]);
+  });
+  return mergedRawGames;
+};
+
+/**
+ * @summary Return a list of games
+ * @function
+ * @returns {Promise<Object[]>} Promise object represents a list of games
+ */
+const getGames = async (query) => {
+  const connection = await conn.getConnection();
+  const round = query ? query.round : null;
+
+  try {
+    const sqlParams = {};
+    if (round) {
+      sqlParams.round = round;
+    }
+    const sqlQuery = `
+    SELECT CN.CARD_NUMBER, CS.SUIT, G.GAME_ID, R.ROUND, G.MAXIMUM_BET, 
+    G.MINIMUM_BET, G.BET_POOL 
+    FROM GAMES G
+    INNER JOIN ROUNDS R ON G.ROUND_ID = R.ROUND_ID 
+    LEFT OUTER JOIN TABLE_CARDS TC ON TC.GAME_ID = G.GAME_ID 
+    LEFT OUTER JOIN CARDS C ON TC.CARD_ID = C.CARD_ID 
+    LEFT OUTER JOIN CARD_NUMBERS CN ON C.CARD_NUMBER_ID = CN.CARD_NUMBER_ID 
+    LEFT OUTER JOIN CARD_SUITS CS ON C.CARD_SUIT_ID = CS.SUIT_ID 
+    ${round ? 'AND ROUNDS.ROUND = :round' : ''}
```

Comment:
I tried `GET /games?round=blind` and I got this error:
```
Error: ORA-00904: "ROUNDS"."ROUND": invalid identifier
```
Could you look into this?